scripts/list-tasks/main.go

```diff
@@ -0,0 +1,157 @@
+// The list-tasks script enumerates all tasks using any variant of a particular
+// connector, and add them to the active flowctl draft.
+//
+// It consults 'https://raw.githubusercontent.com/estuary/connectors/refs/heads/main/<connector>/VARIANTS'
+// to get an updated list of all variant names of a particular connector, and then lists all matching
+// tasks using a 'flowctl raw get' query like:
+//
+//	flowctl raw get --table live_specs --query select=catalog_name,connector_image_name \
+//	  --query 'spec_type=eq.capture' \
+//	  --query 'connector_image_name=in.("ghcr.io/estuary/source-mysql")' \
+//	  --query 'catalog_name=like.acmeCo/*'
+//
+// This is primarily meant for bulk editing workflows, which are described in more detail in
+// 'docs/feature_flags.md'.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"sort"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var scriptDescription = `
+The list-tasks script enumerates all tasks using any variant of a
+particular connector, and optionally add them to the active flowctl
+draft.
+
+Refer to 'docs/feature_flags.md' for more information on the intended
+use-case and workflow.
+`
+
+var (
+	logLevel = flag.String("log_level", "info", "The log level to print at.")
+
+	taskType   = flag.String("type", "capture", "The type of catalog spec to list (typically 'capture' or 'materialization').")
```

Comment:
Do we need this flag? It  seems redundant with the `imageName`, since a given image will exclusively be a capture or a materialization.

Reply:
Yeah, there's probably not much utility in the task type filter option for bulk config edits since practically speaking the connector name needs to be used.

This script grew out of a helper script I wrote a while back for other reasons, where it actually was useful to be able to say like "give me a list of all capture tasks for tenant `acmeCo`" even when I wasn't initially sure what type of database they were capturing from.

I've changed the default value for this setting so that it won't filter on task type at all, but I've left the actual filter option in because I could still see it being useful in some circumstances and it's only like 4 lines of code.